challenge.rb

```diff
@@ -57,7 +79,15 @@ def ac_monitor current_temp, ac_working, desired_temp
 # - return "fizzbuzz" if `num` is evenly divisible by both 3 and 5
 # - return the value of `num` if `num` isn't divisible by either 3 or 5
 
-# Replace this comment with your code.
+def fizz_buzz_calculator num
+  if num % 3 == 0
+    return "fizz"
+  if num % 5 == 0
+    return "buzz"
+  if num % 3 == 0 && num % 5 == 0
+    return "fizzbuzz"
+  else
+    return "num"
```

Comment:
Spot on!! Just watch those missing `end`s. You need one after the `else` and one to end the method.